frontend/components/EnrollSecrets/SecretField/_styles.scss

```diff
@@ -1,5 +1,7 @@
 .secret-field {
   &__secret-input {
+    margin-bottom: 8px;
```

Comment:
This works, too! I guess it's the "boring" solution, which probably makes the most sense to most people.

Reply:
Lol yeah, but code readability is important as well, wouldn't want to have to decipher stylesheets every time I look at one.